File: rnd_number/C/rnd.c
// Compile with gcc rnd.c and run with ./a.out
// Generates and prints a random number from 0 to 1000.

#include <stdlib.h>
#include <stdio.h>
#include <time.h>

void main(int argc, char *argv[]){
    srand((unsigned int)time(NULL)); // Seed or it will generate the same random number on every execution
    int rndNr = rand() % 1000;

    printf("Your random number is: %d\n", rndNr);
}

File: rnd_number/Java/rnd.java
// Use with java rnd.java
// Generates and prints a random number between 1 and 1000
import java.util.Random;

class Rnd
{
    public static void main(String args[])
    {
        Random rand = new Random();
        int rndNr = rand.nextInt(1000) + 1; // We + 1 it to get it between 1 and 1000.

        System.out.println("Your random number is: " + rndNr);
    }
}
